examples/WebApp/webApp.bicep

```diff
@@ -1,7 +1,8 @@
 
-param location string = resourceGroup().location
+param location string = 'EastUS2'
```

Comment:
Was this changed because of the quota limits you were mentioning?

Reply:
Also, keep in mind that if we hard code this our example READ ME no longer is correct when it comes to how to handle the location value.